triggers.md

```diff
@@ -288,6 +288,38 @@ To allow inline speculation rules, use either the `'inline-speculation-rules'` o
 Using `script-src 'inline-speculation-rules'` or `script-src-elem 'inline-speculation-rules'` helps developers to permit inline speculation rules but still disallow unsafe inline JavaScript.
 The `prefetch-src` directive can be used to restrict which URLs can be prefetched or prerendered.
 
+### Eagerness
+
+If developers like to signify how eager they are and how willing they are to tolerate the overhead of a preloading action, they could add an “eagerness” field to the speculation rules.
```

Comment:
This phrasing is slightly strange, because it seems to suggest that signifying eagerness is itself rewarding for developers, as opposed to a means of achieving another goal (efficient use of the user's resources and their own).

It also, to me, seems obvious that "eager" is more eager than "conservative", so I don't think it's necessary to specify.

Maybe something like:

```markdown
Developers can control how eagerly the browser preloads links in order to balance the performance advantage against resource overhead. If not specified, list rules default to `"eager"` and document rules default to `"conservative"`.
```